pkg/agent/datamodel/defaults-controller-manager.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
we don't need this

Reply:
PR to remove this has been sent out: https://github.com/Azure/AgentBaker/pull/194